benchclients/python/benchclients/logging.py

```diff
@@ -1,9 +1,12 @@
 import logging
 
+log = logging.getLogger(__name__)
+
 logging.basicConfig(
-    format="%(levelname)s [%(asctime)s] %(message)s", level=logging.INFO
+    level=logging.INFO,
+    format="%(asctime)s.%(msecs)03d %(levelname)s: %(message)s",
+    datefmt="%y%m%d-%H:%M:%S",
```

Comment:
This takes effect if the benchadapt library used from within `benchclients`.

Reply:
Is benchclients a CLI? Or another set of libraries?

Reply:
Libraries. The only one that's a CLI is benchconnect

Reply:
...and datalogistik, but I think that's still in a separate repo for now

Reply:
Thanks!